packages/edit-site/src/components/navigation-sidebar/navigation-panel/template-navigation-item.js

```diff
@@ -37,7 +39,18 @@ export default function TemplateNavigationItem( { item } ) {
 				onMouseEnter={ () => setIsPreviewVisible( true ) }
 				onMouseLeave={ () => setIsPreviewVisible( false ) }
 			>
-				{ title }
+				<div className="edit-site-navigation-panel__template-item-title">
+					{ title }
+					{ item.type === 'wp_template' &&
+						item.status !== 'auto-draft' && (
+							<Tooltip
+								text={ __( 'Customized' ) }
+								position="top center"
+							>
+								<span className="edit-site-navigation-panel__template-item-customized-dot" />
+							</Tooltip>
```

Comment:
I wonder if the dot alone is just too small to be an effective hover area for the tooltip.
Maybe it could make sense to wrap the entire title? 🤔 

Also the tooltip background seems to me === the sidebar background, making it hard to read.

@jameskoster do you have any opinions?

Reply:
> Maybe it could make sense to wrap the entire title?

That would probably help with the criticism I had below.  Its hard to get it to pop up or even know its there, especially with the cursor already being a pointer in the parent.